Formula/neovim.rb

```diff
@@ -78,8 +78,9 @@ def install
   def caveats; <<-EOS.undent
       The Neovim executable is called 'nvim'. To use your existing Vim
       configuration:
-          ln -s ~/.vimrc ~/.nvimrc
-          ln -s ~/.vim ~/.nvim
+          ln -s ~/.vimrc ~/.config/nvim/init.vim
+          ln -s ~/.vim ~/.config/nvim
+          ln -s ~/.vim/colors ~/.config/nvim/colors
```

Comment:
I wouldn't have thought so either but I tried just the first to and Neovim was unable to find my specified color scheme (`E185: Cannot find color scheme 'mustard'`). 

To double check, I just ran `rm ~/.config/nvim/colors` and opened Neovim. While there is some color highlighting, it again warns me that it cannot find the scheme I specify in my `~/.vimrc`.